aio/aio-proxy/aio_proxy/tests/unit_tests/test_is_siren.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from aio_proxy.search.helpers import is_siren
+
+
+@pytest.mark.parametrize(
+    "siren, expected",
+    [
+        ("123456789", True),
+        ("123 456 689", True),
+        (None, None),
+        ("mother of dragons", False),
```

Comment:
mother of dragons can not get `False` as a response but.. ok.

Reply:
Indeed 😂

Reply:
You could add a siret here as the regex can be tricky between 9 and 14 digits